### PR TITLE
Important fields 1

### PR DESCRIPTION
**Pull request by Mykhailenko**
- fix import bug
 - change important label

### CODE REVIEW

File: rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/nodesource/serialization/NodeSourceConfigurationParser.java [hunk 1]
```diff
@@ -135,7 +138,10 @@ private PluginDescriptor getPluginDescriptor(JSONObject p, String pluginName) {
             String metaType = meta.get("type").isString().stringValue();
             String descr = meta.get("description").isString().stringValue();
             boolean dynamic = meta.get("dynamic").isBoolean().booleanValue();
-            int sectionSelector = (int) meta.get("sectionSelector").isNumber().doubleValue();
+            int sectionSelector = 0;
+            if (meta.get("sectionSelector") != null) {
+                sectionSelector = (int) meta.get("sectionSelector").isNumber().doubleValue();
+            }
```
Comment (MaelAudren) [inline rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/nodesource/serialization/NodeSourceConfigurationParser.java]: Maybe you could replace by:
```
sectionSelector = Optional.ofNullable(meta.get("sectionSelector"))
                                          .map(selector -> selector.isNumber().doubleValue())
                                          .orElse(0);
```

Comment (Mykhailenko) [reply rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/nodesource/serialization/NodeSourceConfigurationParser.java]: Thanks. Perfect

File: rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/nodesource/serialization/NodeSourceConfigurationParser.java [hunk 2]
```diff
@@ -114,13 +109,20 @@ private PluginDescriptor getPolicyPluginDescriptor(JSONObject jsonObject) {
...
-            int sectionSelector = Integer.parseInt(key);
-            String sectionDescription = jsonSectionDescriptions.get(key).isString().stringValue();
-            sectionDescriptions.put(sectionSelector, sectionDescription);
-        }
+        Map<Integer, String> sectionDescriptions = Optional.ofNullable(p.get("sectionDescriptions"))
+                                                           .map(JSONValue::isObject)
+                                                           .map(jsonSectionDescriptions -> {
+                                                               Map<Integer, String> newSections = new HashMap<>();
+                                                               for (String key : jsonSectionDescriptions.keySet()) {
+                                                                   int sectionSelector = Integer.parseInt(key);
```
Comment (aminelouati) [inline rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/nodesource/serialization/NodeSourceConfigurationParser.java]: May move the declaration outside the loop

File: rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/nodesource/serialization/NodeSourceConfigurationParser.java [hunk 3]
```diff
@@ -114,13 +109,20 @@ private PluginDescriptor getPolicyPluginDescriptor(JSONObject jsonObject) {
...
+                                                                   String sectionDescription = jsonSectionDescriptions.get(key)
+                                                                                                                      .isString()
```
Comment (aminelouati) [inline rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/nodesource/serialization/NodeSourceConfigurationParser.java]: here too

File: rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/nodesource/serialization/NodeSourceConfigurationParser.java (hunk 2 above)
Comment (Mykhailenko) [reply rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/nodesource/serialization/NodeSourceConfigurationParser.java]: Thanks, but I use it only inside this loop.

File: rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/nodesource/serialization/NodeSourceConfigurationParser.java (hunk 3 above)
Comment (Mykhailenko) [reply rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/nodesource/serialization/NodeSourceConfigurationParser.java]: also is used only inside the loop

**Pull request merged**